coalib/coala_delete_orig.py

```diff
@@ -26,7 +29,7 @@ def main(section: Section=None):
     not_deleted = 0
     for ofile in orig_files:
         logging.info('Deleting old backup file... '
-                         + os.path.relpath(ofile))
+                     + os.path.relpath(ofile))
```

Comment:
actually I'm not sure about this one...
1. definitely separate commit (also for the change below)
2. could you read up in PEP8 again how this shall be handled? The problem is we seem to allow both styles, and next time some guy changes it back^^ If PEP8 has no opinion on this case, I would rather tell you to revert this change, sorry :/